plugins/mcreator-core/datalists/blocksitems.yaml

```diff
@@ -4074,6 +4153,476 @@
+  type: block
+
+- Blocks.WEATHERED_CUT_COPPER:
+  readable_name: "Weathered Cut Copper"
+  type: block
+
+- Blocks.EXPOSED_CUT_COPPER:
+  readable_name: "Exposed Cut Copper"
+  type: block
+
+- Blocks.CUT_COPPER:
+  readable_name: "Cut Copper"
+  type: block
+
+- Blocks.OXIDIZED_CUT_COPPER_STAIRS:
+  readable_name: "Oxidized Cut Copper Stairs"
+  type: block
+
+- Blocks.WEATHERED_CUT_COPPER_STAIRS:
+  readable_name: "Weathered Cut Copper Stairs"
+  type: block
+
+- Blocks.EXPOSED_CUT_COPPER_STAIRS:
+  readable_name: "Exposed Cut Copper Stairs"
+  type: block
+
+- Blocks.CUT_COPPER_STAIRS:
+  readable_name: "Cut Copper Stairs"
+  type: block
+
+- Blocks.OXIDIZED_CUT_COPPER_SLAB:
+  readable_name: "Oxidized Cut Copper Slab"
+  type: block
+
+- Blocks.WEATHERED_CUT_COPPER_SLAB:
+  readable_name: "Weathered Cut Copper Slab"
+  type: block
+
+- Blocks.EXPOSED_CUT_COPPER_SLAB:
+  readable_name: "Exposed Cut Copper Slab"
+  type: block
+
+- Blocks.CUT_COPPER_SLAB:
+  readable_name: "Cut Copper Slab"
+  type: block
+
+- Blocks.WAXED_OXIDIZED_COPPER:
+  readable_name: "Waxed Oxidized Copper"
+  type: block
+
+- Blocks.WAXED_WEATHERED_COPPER:
+  readable_name: "Waxed Weathered Copper"
+  type: block
+
+- Blocks.WAXED_EXPOSED_COPPER:
+  readable_name: "Waxed Exposed Copper"
+  type: block
+
+- Blocks.WAXED_COPPER_BLOCK:
+  readable_name: "Waxed Copper Block"
+  type: block
+
+- Blocks.WAXED_OXIDIZED_CUT_COPPER:
+  readable_name: "Waxed Oxidized Cut Copper"
+  type: block
+
+- Blocks.WAXED_WEATHERED_CUT_COPPER:
+  readable_name: "Waxed Weathered Cut Copper"
+  type: block
+
+- Blocks.WAXED_EXPOSED_CUT_COPPER:
+  readable_name: "Waxed Exposed Cut Copper"
+  type: block
+
+- Blocks.WAXED_CUT_COPPER:
+  readable_name: "Waxed Cut Copper"
+  type: block
+
+- Blocks.WAXED_OXIDIZED_CUT_COPPER_STAIRS:
+  readable_name: "Waxed Oxidized Cut Copper Stairs"
+  type: block
+
+- Blocks.WAXED_WEATHERED_CUT_COPPER_STAIRS:
+  readable_name: "Waxed Weathered Cut Copper Stairs"
+  type: block
+
+- Blocks.WAXED_EXPOSED_CUT_COPPER_STAIRS:
+  readable_name: "Waxed Exposed Cut Copper Stairs"
+  type: block
+
+- Blocks.WAXED_CUT_COPPER_STAIRS:
+  readable_name: "Waxed Cut Copper Stairs"
+  type: block
+
+- Blocks.WAXED_OXIDIZED_CUT_COPPER_SLAB:
+  readable_name: "Waxed Oxidized Cut Copper Slab"
+  type: block
+
+- Blocks.WAXED_WEATHERED_CUT_COPPER_SLAB:
+  readable_name: "Waxed Weathered Cut Copper Slab"
+  type: block
+
+- Blocks.WAXED_EXPOSED_CUT_COPPER_SLAB:
+  readable_name: "Waxed Exposed Cut Copper Slab"
+  type: block
+
+- Blocks.WAXED_CUT_COPPER_SLAB:
+  readable_name: "Waxed Cut Copper Slab"
+  type: block
+
+- Blocks.LIGHTNING_ROD:
+  readable_name: "Lightning Rod"
+  type: block
+
+- Blocks.POINTED_DRIPSTONE:
+  readable_name: "Pointed Dripstone"
+  type: block
+
+- Blocks.DRIPSTONE_BLOCK:
+  readable_name: "Dripstone Block"
+  type: block
+
+- Blocks.CAVE_VINES:
+  readable_name: "Cave Vines"
+  type: block
+
+- Blocks.CAVE_VINES:

```

Comment:
I think this was the conversation origin, here there are two entires with a same name